cas/build.gradle

```diff
@@ -74,11 +83,63 @@ task generateKeys {
   }
 }
 
-task runit {    
+task runit {
     doLast{
-        javaexec { 
+        javaexec {
             main="-jar";
-            args = [ "build/libs/cas.war" ]
+            jvmArgs = [
+                // Reduce the memory usage
+                "-Xmx2048M"
+                ]
+            args = [
+                "build/libs/cas.war",
+                "--logging.config=file:///home/cas/etc/cas/config/log4j2.xml",
+                "--spring.cloud.config.server.native.searchLocations=file:///home/cas/etc/cas/config"
```

Comment:
As well this config, with the copy, forces the user to run a build anytime it has to change the cas.properties, which "--spring.cloud.config.server.native.searchLocations=" avoids